src/theme.cpp

```diff
@@ -515,8 +517,16 @@ theme::menu::menu(std::size_t sw, std::size_t sh, const config& cfg)
 	, overlay_(cfg["overlay"])
 	, items_()
 {
-	for(const auto& item : utils::split(cfg["items"])) {
-		items_.emplace_back("id", item);
+	if(resources::controller == nullptr || !resources::controller->is_networked_mp()) {
+		for(const auto& item : utils::split(cfg["items"])) {
```

Comment:
For loop could be moved outside the if block.

Reply:
It runs faster this way.